builder/azure/common/client/config.go

```diff
@@ -224,9 +224,9 @@ func (c Config) Validate(errs *packersdk.MultiError) {
 		if err != nil {
 			errs = packersdk.MultiErrorAppend(errs, fmt.Errorf("client_jwt is not a JWT: %v", err))
 		} else {
-			if claims.ExpiresAt < time.Now().Add(5*time.Minute).Unix() {
-				errs = packersdk.MultiErrorAppend(errs, fmt.Errorf("client_jwt will expire within 5 minutes, please use a JWT that is valid for at least 5 minutes"))
-			}
+			//if claims.ExpiresAt < time.Now().Add(5*time.Minute).Unix() {
```

Comment:
OIDC Access Tokens have a shorter validatity, so throwing this error prevents proper OIDC support, removing it fixes this, I have a slack link with some more context from the provider folks if needed